app/controllers/memberships_controller.rb

```diff
@@ -1,6 +1,6 @@
 class MembershipsController < ApplicationController
   before_action :authenticate_person!, only: :index
-  after_action :verify_authorized, only: :index
+  after_action :verify_authorized, only: [:index, :approval]
```

Comment:
Maybe blocking:  Do we also want to verify scoped for all actions? See: https://github.com/varvet/pundit#ensuring-policies-and-scopes-are-used